spalp_example/src/EventSubscriber/SpalpExampleConfigLocationAlterSubscriber.php

```diff
@@ -34,8 +34,8 @@ public function doConfigLocationAlter(SpalpConfigLocationAlterEvent $event) {
       // TODO: proper dependency injection example.
       $module_path = drupal_get_path('module', 'spalp_example');
       $locations = [
-        'config' => $module_path . '/spalp_example.config.json',
-        'schema' => $module_path . '/spalp_example.config.schema.json',
+        'config' => $module_path . '/config/spalp/spalp_example.config.json',
```

Comment:
thanks - I should have sorted this out at the time of making the location change

Reply:
odd, this is already correct:
https://github.com/Capgemini/spalp/blob/8.x-1.x/spalp_example/src/EventSubscriber/SpalpExampleConfigLocationAlterSubscriber.php

You must have an outdated version of the code